time/src/main/java/io/spine/time/LocalDateTemporal.java

```diff
@@ -20,30 +20,29 @@
 
 package io.spine.time;
 
-import java.util.concurrent.TimeUnit;
+import com.google.protobuf.Timestamp;
+import io.spine.time.temporal.TemporalMessage;
```

Comment:
Why do we need the `temporal` sub-package, assuming that all the implementations are in the `time` package?

Reply:
We want to avoid package splitting, as `Temporal` belongs to `base` and not to this project.

Reply:
Can we have `Temporal` next to `Time` then?